src/taco-cli/cli/run.ts

```diff
@@ -186,7 +190,7 @@ class Run extends commands.TacoCommandBase {
             name: "local",
             run: Run.local,
             canHandleArgs(commandData: commands.ICommandData): boolean {
-                return !!commandData.options["local"];
+                return !!commandData.options["local"] || !!commandData.options["livereload"] || !!commandData.options["devicesync"];
```

Comment:
Agreed. this is not the right place to do it. I'm currently reworking the code to a better structure given what I've just learned.